src/suggestEnvironmentVars.ts

```diff
@@ -0,0 +1,121 @@
+import * as vscode from "vscode";
+import { execSync } from "child_process";
+
+import triggers from "./triggers";
+let cursorPosition = 0;
+
+export function getEnvsFromEnvCommand(): {} {
+  const editor = vscode.window.activeTextEditor;
+  if (!editor) {
+    return {};
+  }
+
+  const l2FilePath = editor.document.fileName;
+  if (!l2FilePath.endsWith(".l2")) {
+    return {};
+  }
+
+  try {
+    // const commandOutput = execSync(`./build/l2 --env ${l2FilePath}`, { cwd: "/home/lovestaco/repos/Lama2", }).toString(); // For local debugging 
+    const commandOutput = execSync(`l2 --env ${l2FilePath}`).toString();
+
+    const mapStartIndex = commandOutput.indexOf("{");
```

Comment:
Ideally, we don't want to rely on this. Say, if a logger has a `{` character, this is going to break. So, if at all possible we have a full quiet mode in the command output itself, so that there's no need to find the map's starting point. We should simply be able to do `JSON.parse` on incoming string.

Reply:
Yes made the change